ugandaemr-304.md

```diff
@@ -0,0 +1,55 @@
+## UgandaEMR 3.0.4 Release Notes
+
+### New Features
+1.  Patient Flags
+    * Due for third DNA PCR
+    * Overdue for third DNA PCR
+    * Updated Criteria for Due and Overdue for Second DNA PCR
+ 2. HIV MER Indicator Reports 2.4 
+    * TX_ML Report
+ 3. Data Extraction Reports
+    * Postnatal Register
+    
+    
+  
+### Fixes Made
```

Comment:
@solemabrothers Remove all the spaces before the numbers in this section, all should begin not indented